Server/Actions/ActionObject.cs

```diff
@@ -1,8 +1,10 @@
-﻿using Server.Elements;
+﻿using Server.Base;
+using Server.Database;
+using Server.Elements;
 
 namespace Server.Actions
 {
-    public class ActionObject
+    public abstract class ActionObject
```

Comment:
I can't imagine why someone would want the base ActionObject, so making this abstract seemed okay.

Reply:
Agreed, should have been abstract from the start.